include/glow/Graph/Utils.h

```diff
@@ -132,6 +132,21 @@ class GraphPreOrderVisitor : public PreOrderVisitor {
   }
 };
 
+/// Change the type to \p newTy for all the results of \p node
```

Comment:
What happens if the original type is Integer?  Is the only conversion possible is between Float and FP16? In that case, why is this a general API of the graph?

Reply:
The API would work equally for integer. Why do you think this is not general?

Note: I didn’t implemented the underlying casts from fp to int because I didn’t have any use case, but I don’t see any problem with extending it for that.